code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -259,6 +280,10 @@
 		if(length(hive.xenos_by_typepath[/mob/living/carbon/xenomorph/hivemind]))
 			to_chat(src, span_warning("There cannot be two manifestations of the hivemind's will at once."))
 			return
+	else if(new_caste_type == /mob/living/carbon/xenomorph/king)
```

Comment:
could be one if

Reply:
there's some other else stuff there so I'mma just leave it for now.